features/step_definitions/brief_response_steps.rb

```diff
@@ -21,6 +21,9 @@
 Given /^I am logged in as the buyer of a closed brief$/ do
   closed_brief = get_briefs('digital-outcomes-and-specialists-2', 'closed').sample
   @buyer = closed_brief['users'][0]
```

Comment:
Yeah this should be cool. Could have unexpected consequences if we use both
`I am logged in as the buyer of a closed brief`
`I am logged in as the buyer of a closed brief with responses`
in the same test anywhere at the moment but seems sensible.

Reply:
Ahhh, interesting :o. We could actually refactor them into a single step with "with responses" optional... hmmm :d